.github/workflows/pre-commit.yml

```diff
@@ -0,0 +1,14 @@
+name: pre-commit
+
+on:
+  pull_request:
+  push:
+    branches: [main]
```

Comment:
This repository doesn't have a `main` branch.

Reply:
Oh, right.